src/tests/JIT/HardwareIntrinsics/Arm/Shared/Helpers.cs

```diff
@@ -5218,6 +5218,33 @@ public static float MultiplyExtended(float op1, float op2)
             }
         }
 
+        public static float TrigonometricMultiplyAddCoefficient(float op1, float op2, byte imm)
```

Comment:
shouldn't there be `double` equivalent of this helper?

Reply:
The `double` version is further down `Helpers.cs`, on line 5292.